source/fe/fe_bernardi_raugel.cc

```diff
@@ -55,7 +55,25 @@ FE_BernardiRaugel<dim>::FE_BernardiRaugel(const unsigned int p)
 
   // const unsigned int n_dofs = this->dofs_per_cell;
 
-  this->mapping_type = mapping_none;
+  if(dim==2)
+    this->mapping_type = {mapping_none, mapping_none, 
+                          mapping_none, mapping_none,
+                          mapping_none, mapping_none, 
+                          mapping_none, mapping_none,
+                          mapping_piola, mapping_piola,
+                          mapping_piola, mapping_piola};
```

Comment:
I believe you'll have to update that, right?

Reply:
Yes, I will revert it to `{mapping_none}` once the fill_fe_values overload is finished.